idaes/config.py

```diff
@@ -249,6 +249,44 @@ def _new_idaes_config_block():
         ),
     )
 
+    cfg.declare(
+        "ipopt_l1",
+        pyomo.common.config.ConfigBlock(
+            implicit=False,
+            description="Default config for 'ipopt' solver",
```

Comment:
Should all these descriptions reference `ipopt_l1` instead of `ipopt`?

Reply:
Yeah.  I'll fix it.